src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1524,7 +1536,9 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
     std::vector<Note*> downStemNotes;
     int upVoices       = 0;
     int downVoices     = 0;
-    double nominalWidth = ctx.conf().noteHeadWidth() * staff->staffMag(tick);
+    // Fret width plus white background box for TAB
+    double nominalWidth = !isTab ? ctx.conf().noteHeadWidth() * staff->staffMag(tick)
+                          : (ctx.conf().fretWidth(staffType) + 0.2 * staff->spatium(tick)) * staff->staffMag(tick);
```

Comment:
What is 0.2? Is that the white background padding around the fret number? If so, it would be better to properly define/retrieve this value (I think I may have seen it defined somewhere, it definitely should be a style definition if it isn't already)

Reply:
It is.  It isn't defined properly anywhere.  I'll add it as a style value.